static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,30 @@ define([
         }
     }
 
+    function paidforCardLoaded(el) {
+        var height,
+            $adSlot = $(el);
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+            });
+
+            fastdom.write(function () {
+                $('.js-most-popular-footer .tabs__pane').css('height', height);
+            });
+        }
+
+        createToggle(el);
+    }
+
     CommercialComponent.prototype.postLoadEvents = {
         bestbuy: function (el) {
             new Tabs().init(el);
         },
         capi: createToggle,
         capiSingle: createToggle,
-        paidforCard: createToggle
+        paidforCard: paidforCardLoaded
```

Comment:
You are right. I had problems with naming it. But that function does not only change height. It is more general because it also initialise tabs. I couldn't think of any short name.